startupscript/butane/001-install-node.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# install-node.sh installs Node.js on the VM.
```

Comment:
These scripts are called as shell scripts (not sourced?). Just checking as we wouldn't want to "set -e" if it is a sourced-script.

These should all be standardized to set:

```
set -o errexit
set -o nounset
set -o pipefail
```

Even if they are simple and don't actually include environment variable references. Don't put it on a maintainer to have to add it.

Reply:
no they are not sourced